cmd/zpool/zpool_main.c

```diff
@@ -207,7 +207,7 @@ static const char *
 get_usage(zpool_help_t idx) {
 	switch (idx) {
 	case HELP_ADD:
-		return (gettext("\tadd [-fn] [-o property=value] "
+		return (gettext("\tadd [-fgLnp] [-o property=value] "
```

Comment:
Why is "n" in between "L" and "p"? Also, should gLp always go last, always go first, or float around as needed?